packages/xcoobee-cookie-kit-react/src/CookieKitContainer.js

```diff
@@ -418,6 +419,15 @@ export default class CookieKitContainer extends React.PureComponent {
             />
           </React.Fragment>
         )}
+        <BridgeCommunicator
+          ref={(bridgeRef) => {
+            this.bridgeRef = bridgeRef;
+          }}
+          campaignReference={campaignReference}
+          onCookieOptionsLoad={cookieOptions => this.resolveConnectedCookieConsents(cookieOptions)}
+          onLoginStatusChange={status => this.onLoginStatusChange(status)}
```

Comment:
```suggestion
          onLoginStatusChange={this.onLoginStatusChange}
```

Reply:
I think you may have missed this comment. I think it may have be hidden behind a collapsed section that GitHub added.